ONSdigital/response-operations-ui
# [TBL] Call party direct when searching respondents by email

# Motivation and Context

Removes the call to backstage for searching for a respondent by email address.

# What has changed



Changes the `search_respondent_by_email` method in respondent_controllers.py to hit the party service directly.

# How to test?



Run the unit tests, make sure they pass.
Tail the logs for backstage, party service. Hit the `/respondents/` page and search by email. Make sure backstage doesn't get the call, but party service does.
